client/my-sites/checkout/checkout/secure-payment-form.jsx

```diff
@@ -605,13 +610,28 @@ export class SecurePaymentForm extends Component {
 
 	renderGreatChoiceHeader() {
 		const { translate } = this.props;
-		const headerText = translate( 'Great choice! How would you like to pay?' );
 
-		this.props.setHeaderText( headerText );
+		if ( 'variant' === abtest( 'checkoutSealsCopyBundle' ) ) {
+			const headerText = translate( 'Confirm Your Order' );
+			const subHeaderText = translate(
+				'Review your order and proceed with a payment method to finish'
+			);
+
+			this.props.setHeaderText( headerText, subHeaderText );
+		} else {
+			const headerText = translate( 'Great choice! How would you like to pay?' );
+			this.props.setHeaderText( headerText );
+		}
 	}
 
 	render() {
-		const visiblePaymentBox = this.getVisiblePaymentBox( this.props );
+		const visiblePaymentBox = this.getVisiblePaymentBox( this.props ),
+			moneyBackGuaranteeSeal =
```

Comment:
I think we could move this `moneyBackGuaranteeSeal ` line to [setInitialPaymentDetails()](https://github.com/Automattic/wp-calypso/blob/db87fd89d486ca083690dcacd791b683e08d235f/client/my-sites/checkout/checkout/secure-payment-form.jsx#L94), similar to `shouldUseStripeElements` assignment [here](https://github.com/Automattic/wp-calypso/blob/db87fd89d486ca083690dcacd791b683e08d235f/client/my-sites/checkout/checkout/secure-payment-form.jsx#L103). This way, we can avoid repeating the assignment of `moneyBackGuaranteeSeal ` in each of the payment method's components.

So in `setInitialPaymentDetails()`:
```
this. moneyBackGuaranteeSeal = ! hasOnlyDomainProducts( this.props.cart ) &&
				'variant' === abtest( 'checkoutSealsCopyBundle' ) &&
				indexOf( [ 'credits', 'free-trial', 'free-cart' ], visiblePaymentBox ) === -1;
```

Then, for each of the payment method components, we could pass this. moneyBackGuaranteeSeal as a prop.
e.g.: 
```
<WechatPaymentBox
    cart={ this.props.cart }
    transaction={ this.props.transaction }
    selectedSite={ this.props.selectedSite }
    redirectTo={ this.props.redirectTo }
    moneyBackGuarantee={ this. moneyBackGuarantee }
    presaleChatAvailable={ this.props.presaleChatAvailable }
>
    { this.props.children }
</WechatPaymentBox>
```

Reply:
Very good point! As much as I agree with your suggestion (and even started working on it :D) I think we should leave this for a future refactoring in the hypothetical scenario where this gets implemented as new control.